src/OrleansTestingHost/OrleansTestingHost.csproj

```diff
@@ -96,11 +101,25 @@
     <None Include="packages.config" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
+  <!-- Begin Orleans: Without these lines the project won't build properly -->
```

Comment:
This is just a boiler-plate csproj comment (junk) which was replaced by the codegen logic below.  I added codegen to this project because it now has grains.